### DISCUSSION TITLE
How do I deploy a Tableau website?

### DISCUSSION THREAD

**Discussion by sylvesterroos** [elixir-tools/tableau#73] [Q&A] [answered]
Probably a dumb question, but I can't seem to figure it out. Do I just `MIX_ENV=prod mix tableau.server`? If so, how do I disable live reloading, since there won't be any file changes?

**Comment by mhanberg** [comment, marked as answer]
Generally you use a CDN like Netlify. 

You configure Netlify to run `mix tableau.build` and then it uploads the built site to its network and serves that. 

mix tableau.server is just for local dev. 

Other folks have also used GitHub pages, but I personally am not sure how to set that up.

**Reply by sylvesterroos** [reply]
Yup, got it working, thanks!

I tried simply opening the `index.html` to see if it worked, but the formatting was missing. Only after running it on Nginx did I get the expected result
